### PR TITLE
Set environment name before activation

### LINKED ISSUE #951: Offer option to name the default environment via UI before initialization
Clones created from the original sites always have names ('wp vp clone **--name=xyz**') but the original one doesn't – we call it **default** (previously master, see #847). When browsing the environments – see https://github.com/versionpress/versionpress/pull/858 – the name `default` doesn't look too good next to better env. names like `staging`, `dev` etc. We should give the user an opportunity to provide a better name for the default environment, for example, some users might call their original environment `production` or something like that.

### PR DESCRIPTION
**Pull request by borekb**
Resolves #951.

There's a new piece of UI on the pre-activation screen:

name-environment-pre-activation

The entered environment name is validated slightly in JavaScript (quick alert, nothing too fancy) and then on the server-side. If server-side validation fails, `default` is used as an env name (see #847).

Reviewers:

- [x] @JanVoracek 
- [x] @octopuss

### CODE REVIEW

File: plugins/versionpress/admin/inc/activationPanel.php [hunk 1]
```diff
@@ -79,6 +104,25 @@
...
 
+                <h4>Name your environment</h4>
+
+                <ul>
+                    <li>
+                        <span class="dashicons dashicons-info icon" style="color: #5b9dd9; font-size: 1.2em; left: -28px;"></span>
+                        E.g., <code>production</code>, <code>dev</code> or <code>my-machine</code>. This will help you identify this environment later.
+
+                        <div style="margin-top: 10px;">
+                            <label for="envname"><?php _e('Environment name:') ?></label>
```
Comment (octopuss) [inline plugins/versionpress/admin/inc/activationPanel.php]: Localization function call is not necessary right now.

File: plugins/versionpress/admin/inc/activationPanel.php [hunk 2]
```diff
@@ -7,6 +7,31 @@
 
 ?>
 
+<style>
+    h4 {
+        margin-top: 30px;
+        margin-bottom: 10px;
+    }
+</style>
```
Comment (octopuss) [inline plugins/versionpress/admin/inc/activationPanel.php]: Move styles into `styles.css`.

File: plugins/versionpress/admin/inc/activationPanel.php (hunk 1 above)
Comment (borekb) [inline plugins/versionpress/admin/inc/activationPanel.php]: Done.

File: plugins/versionpress/admin/inc/activationPanel.php (hunk 2 above)
Comment (borekb) [inline plugins/versionpress/admin/inc/activationPanel.php]: Done.

**Pull request merged**